runtime/libia2/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 4.0)
 project(libia2)
 
-add_library(libia2 ia2.c init.c threads.c main.c exit.c memory_maps.c thread_name.c)
+add_library(libia2 ia2.c init.c threads.c main.c memory_maps.c thread_name.c)
```

Comment:
TODO: Do we want to keep the option of exiting the process in the untrusted compartment?